### DISCUSSION TITLE
Make use of fontkit.js

### DISCUSSION THREAD

**Discussion by ollimeier** [Lorp/samsa#83] [Ideas]
Hi @Lorp, 

have you ever thought about using https://github.com/foliojs/fontkit for processing fonts? I found it very useful. I used it for fontspecimen.com

Olli

**Comment by Lorp** [comment]
I decided at the outset to make Samsa entirely independent of FontKit and OpenType.js. Those projects are great, but on the whole I’m pleased that Samsa is free of dependencies. Optimized code for variation calculation isn’t ideal for providing “hooks” for visualization, which is essentially Samsa’s _raison d’être_.

Thanks for reminding me about fontspecimen.com — it’s excellent work, and FontKit sounds ideal.
